### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by jorma16**
This PR adds the necessary dockerfile to run this project as a docker container with dynamic configuration.

In the README.md I explain how it works.

I also restructured the project structure to group the screenshots in a folder.

### CODE REVIEW

**Comment by randallm** [comment]
Hi @jorma16, thank you for your contribution. Looks good to me, but before we merge could you squash your docker related commits together?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM node:8.1.2
+
+EXPOSE 4567
+
+RUN mkdir -p /opt/arena
+WORKDIR /opt/arena
+COPY package.json /opt/arena
+RUN npm --no-color install --production
```
Comment (spencer-brown) [inline Dockerfile]: Should this use `yarn` @randallm ?

**Comment by jorma16** [comment]
Like that? Thanks in advice!

**Comment by pballester** [comment]
there are any plans for upload an official image to the docker hub registry? That would be great!  :wink:

File: Dockerfile (hunk 1 above)
Comment (randallm) [inline Dockerfile]: While we use yarn for development purposes (and part of our contributor guidelines), it's not strictly necessary to deploy Arena in production. So I'm slightly in favor of not enforcing yarn here, but don't feel particularly strongly about it. What are your thoughts?

**Comment by randallm** [comment]
@jorma16 

> Like that? Thanks in advice!

Close! Could you combine commits 302c631, e4120fd, and 3ba997d using an [interactive rebase](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history) or other method of rewriting git history, so that this PR is the minimum number of commits necessary? That makes it easier to bisect in the event with have to debug later. Additionally, feel free to clean up this branch and force push so that we don't have a squashed commit _in addition to_ the existing commits. Just trying to keep things clean here :)

@pballester 

> there are any plans for upload an official image to the docker hub registry?

This is something we are interested in doing. I will look into it in the coming days. Thank you for the suggestion!

File: Dockerfile (hunk 1 above)
Comment (spencer-brown) [inline Dockerfile]: it makes sense to me that we'd use `yarn` in the `Dockerfile` to ensure consistency between live instances of `arena` and dev instances of `arena`.

**Comment by jorma16** [comment]
Now the squash is well done, thank you @randallm

**Comment by jorma16** [comment]
What is the state of this?

**Comment by randallm** [comment]
Merged. Thank you for your contribution @jorma16. This will make it into the next version, followed by official docker hub support.

**Comment by jorma16** [comment]
Hi guys! What is the state of the official docker hub repository? We need this to deploy the frontend in a container and we would like to use the official repository.

Thanks!

**Comment by randallm** [comment]
Hi @jorma16, I have published Arena v1.4.1 (the latest) to Docker Hub:

https://hub.docker.com/r/mixmaxhq/arena/tags/

You can pull with `docker pull mixmaxhq/arena:v1.4.1`. Please let me know if you have any other questions or comments :)

**Pull request merged**